unified-runtime/source/loader/ur_adapter_registry.hpp

```diff
@@ -274,6 +274,19 @@ class AdapterRegistry {
           continue;
         }
       }
+      // Skip legacy L0 adapter if the v2 adapter is requested, and vice versa.
+      if (std::string(adapterName).find("level_zero") != std::string::npos) {
+        auto v2Requested = getenv_tobool("UR_LOADER_USE_LEVEL_ZERO_V2", false);
+        auto v2Adapter =
+            std::string(adapterName).find("v2") != std::string::npos;
+
+        if (v2Requested != v2Adapter) {
+          logger::info("The adapter '{}' is skipped because {} is{}set.",
```

Comment:
```suggestion
          logger::info("The adapter '{}' is skipped because {} is {} set.",
```

Reply:
This is actually intentional because if I add spaces around '{}', then I will have 'is  set' (with 2 spaces) when v2Requested is true.

Reply:
oh, I see :D Does {} add a space of its own? the condition is `v2Requested ? " " : " not "`. Wouldn't it possible to just do `v2Requested ? "" : "not"` (without extra spaces)? Seems simpler that way.

Reply:
No, it does not add spaces by it's own. I changed it to just have conditional 'is set' or 'is not set' so it looks more natural.